.github/workflows/container.yml

```diff
@@ -0,0 +1,23 @@
+name: container
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build-container:
+    runs-on: ubuntu-latest
+    name: build_container
+    steps:
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: TierMobility
```

Comment:
Not sure if `username: TierMobility` is right.

Reply:
Looks like it's working :)